build.gradle

```diff
@@ -52,6 +52,8 @@ ext {
     openCsvVersion = '4.1'
 
     qrScannerVersion = '1.9.8'
+    journeyappsZxingScannerVersion = '3.6.0'
```

Comment:
What's the difference with what we already have. What's the added benefit ?

Reply:
+1

Reply:
Currently the viewfinder freezes during scan, I tried removing all the network calls to check if that might be the reason for the freezing of the camera however I still found slight jerks and freezing of the viewfinder whenever a barcode was detected. Therefore I used this new library which is smooth and the viewfinder doesn't freeze like it did previously. At the time of finalizing this PR I will remove the older dependency from our code completely.